lib/reporters/mini.js

```diff
@@ -191,6 +196,22 @@ MiniReporter.prototype.finish = function (runStatus) {
 		});
 	}
 
+	if (this.failingCount > 0) {
```

Comment:
Since the known failure count is shown after the number of passing tests, perhaps output these *before* the actual failures.